ReSharper.FSharp/test/src/FSharp.Tests/QuickFixes/NamespaceToModuleTest.fs

```diff
@@ -10,14 +10,8 @@ type NamespaceToModuleTest() =
     inherit QuickFixTestBase<NamespaceToModuleFix>()
 
     override x.RelativeTestDataPath = "features/quickFixes/namespaceToModule"
+    
+    override x.OnQuickFixNotAvailable(_, _) = Assert.Fail(ErrorText.NotAvailable);
```

Comment:
Let's do it in `FSharpQuickFixTestBase`? It'll later allow us to clean some other tests too.